python/observability/conf/environment/example-override.yaml

```diff
@@ -0,0 +1,23 @@
+model:
+  id: "test-model"
+  type: "BINARY_CLASSIFICATION"
+  version: "0.1.0"
+  schema:
+    prediction_column: "accept"
+    prediction_column_type: "BOOLEAN"
+    timestamp_column: "request_timestamp"
+    feature_columns:
+      - "distance"
+      - "transaction"
+    feature_column_types:
+      - "INT32"
+      - "FLOAT64"
+arize:
```

Comment:
Although it's unlikely we will provide integrations with multiple providers any time in the near future, shall we design the configuration and classes such that an `observability_backend` is taken into account? Eg:

```yaml
observability_backend:
    type: arize
    arize_config:
        api_key: "SECRET_API_KEY"
        space_key: "SECRET_SPACE_KEY"
```